travis/test_server.py

```diff
@@ -541,6 +541,7 @@ def main(argv=None):
                     'coverage', 'run', '--stop-after-init',
                     '--test-enable', '--init', None,
                     '--log-handler', 'openerp.tools.yaml_import:DEBUG',
+                    'openerp.models.schema:DEBUG',
```

Comment:
@moylop260 
Sorry for my short description 

This change is for avoid the follow error 

```
Testing mg:
createdb: database creation failed: ERROR:  database "openerp_test" already exists
/home/odoo/odoo-saas-17/odoo-bin -d openerp_test --db-filter=^openerp_test$ --log-level info openerp.models.schema:DEBUG --pidfile=/tmp/odoo.pid
Usage: odoo-bin [options]

odoo-bin: error: unrecognized parameters: 'openerp.models.schema:DEBUG'
```